post_install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+OS_NAME=$TRAVIS_OS_NAME
+if [ "$OS_NAME" = "" ]; then
+  if [ $(uname -s) = "Darwin" ]; then
+    export OS_NAME=osx
+  else
+	  export OS_NAME=linux
+  fi
+fi
+
+if which xsltproc >/dev/null; then
+  if which wkhtmltopdf >/dev/null; then
+    @echo "installed"
+    #exit
+  fi
+fi
+
+if [ "$OS_NAME" = "linux" ]; then
+  sudo apt-get update -y
+  sudo apt-get install -y libxml2 xsltproc wget xfonts-75dpi
```

Comment:
Want me to remove the install steps in our Actions config? This file might be better called `install_deps.sh`

https://github.com/moov-io/1120x/blob/811e72e7b11546ccb4d71407dc71e4d3c808254c/.github/workflows/go.yml#L20-L24

Reply:
thanks, will use install_deps.sh